### PR TITLE
Update test tolerance ahead of merging PR #227

### PR DESCRIPTION
**Pull request by bjacob**

### CODE REVIEW

File: ruy/test.h [hunk 1]
```diff
@@ -1401,15 +1401,19 @@ bool Agree(ExternalPath external_path1, const Matrix<Scalar>& matrix1,
       // 1 from using different rounding in right shifts, so the tolerance on
       // the difference may have to be as large as 2.
       tolerated_max_diff = 2;
-    } else if (RUY_PLATFORM_ARM || RUY_PLATFORM_X86) {
+    } else if (RUY_PLATFORM_ARM) {
       // All our code paths on ARM and x86, including SIMD paths, are bit-exact
```
Comment (silvasean) [inline ruy/test.h:1405]: remove mention of x86 here.

File: ruy/test.h [hunk 2]
```diff
@@ -1401,15 +1401,19 @@ bool Agree(ExternalPath external_path1, const Matrix<Scalar>& matrix1,
...
+      // TODO: update the x86 path to adapt to that and reset that tolerance
+      // to 0.
+      tolerated_max_diff = 1;
     } else {
-      // In this case, we are comparing ruy paths only, but outside of ARM or
-      // x86. At the moment, it so happens that ruy's x86 SIMD code paths are
-      // bit exact with reference, but we allow for some tolerance on other
-      // architectures.
+      // Other architectures, which we don't have dedicated code paths for
+      // at the moment - TODO: try resetting that tolerance to 0, since by
```
Comment (silvasean) [inline ruy/test.h:1415]: start TODO at beginning of line.

**Review by silvasean — APPROVED** [review]

File: ruy/test.h (hunk 1 above)
Comment (bjacob) [reply ruy/test.h]: done, thanks!

File: ruy/test.h (hunk 2 above)
Comment (bjacob) [reply ruy/test.h]: done, thanks!

**Pull request closed without merging**